dicom_utils/container/output.py

```diff
@@ -204,7 +204,7 @@ def __call__(self, inp: Union[Input, Dict[str, RecordCollection]]) -> Dict[str,
             if self.by_case:
                 f.write(f"{name}\n")
             else:
-                files = [Path(*rec.path.parts[-2:]) for rec in collection]
+                files = [Path(*rec.path.parts[-4:]) for rec in collection]
```

Comment:
Having a named constant for this `-4` would be helpful for interpret-ability.

Reply:
Moved to a new function with explanation